WordPress/Classes/Networking/Plugin Management/JetpackPluginManagementClient.swift

```diff
@@ -0,0 +1,55 @@
+import Foundation
+import WordPressKit
+
+class JetpackPluginManagementClient: PluginManagementClient {
```

Comment:
This isn't in WPKit because I need the reference to `JetpackSiteRef`. If not for that this would live there.